Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_++.cpp

```diff
@@ -141,6 +145,44 @@ int main(int argc, char** argv)
 			{
 				SPDLOG_ERROR("Missing value for flag -port");
 			}
+			else if (next_arg.compare("-caffe") == 0 && (i + 1) < argc)
```

Comment:
Also, else if into a continue statement is redundant. It's either else if into nothing at the end, or if into a continue.
I changed it to the if-continue style in when I cleaned up the service module 😄, see [Sight++.cpp on Dev](https://github.com/SightPP/Intel_IXN_SightPP/blob/Development/Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_%2B%2B.cpp)